### DISCUSSION TITLE
Re-add in Centos Base repos for airgapped grid?

### DISCUSSION THREAD

**Discussion by scarl3t0** [Security-Onion-Solutions/securityonion#7462] [Unsupported Versions] [answered]
Hello All,

I have an Airgap installation of Security Onion for the simple fact that Github is not allowed within my organization. Our vulnerability scanner identifies missing package updates periodically despite being on the latest version of SO. I have tested adding in the base CentOS repo to one of the servers for OS updates and unsing yum update to get the packages, which does seem to work for a short time and doesn't break SO. The repo is unfortunately overwritten a short time later by some function that I have yet to identify.

My question is, is there a better way to do this? Should I switch off Airgap and just add in the base repo, then still use the ISO for versioned updates? Or is there an individual function I could disable to prevent this overwriting so I can use yum-cron to get weekly updates on a schedule?

TYIA!

**Comment by dougburks** [comment, marked as answer]
Our current supported methods are:
- If you perform an airgap installation, then all updates are applied using the latest ISO image.
- If you perform a non-airgap installation, then you will need Internet access.

You may be able to make other options work, but it's not tested or supported.

Also note that some vulnerability scanners only look at package version numbers and don't take into account that Redhat (and thus CentOS) quite often backports security fixes without incrementing the version number.

**Reply by scarl3t0** [reply]
Thank you @dougburks! Our organization is a stickler for anything and everything the vulnerability scanner finds. I'll try to stick with a supported method so as not to break things. I appreciate it!
